src/scss/components/card/_basic-card.scss

```diff
@@ -0,0 +1,45 @@
+@charset "UTF-8";
+
+.ptype-stanford-card {
+
+    // Card display
+    .su-card {
+        border: none;
+        box-shadow: none;
+
+        .image {
+            img,
+            figure {
+                display: block;
+                min-height: 397px;
+                width: 100%;
+                object-fit: cover;
+            }
+        }
+
+        section.su-card__contents {
+            padding: 4rem 0 0 0;
+
+            div.su-card__superhead {
+                @include slab;
+
+                font-size: 1.9rem;
+                font-weight: 400;
+                margin-bottom: 0;
+                line-height: 120%;
+              }
+
+            h2,
+            h3 {
+                @include thub-type-b;
+
+                margin-bottom: 0;
+            }
+
```

Comment:
<img width="740" alt="image" src="https://github.com/SU-SWS/taiwan_hub_subtheme/assets/34019925/8157eddf-cfc7-4ad7-9272-02662fe93f87">


@jenbreese everything looks great! It looks like all we're missing is the spacing for the `<p>` tags. We could add a margin-bottom to the heading and add a margin-bottom to the last `<p>` tag 😄 Let me know what you think!

Reply:
@rebeccahongsf I added margins to the `<p>` because headings can be optional. Thank you for catching this!

Reply:
Thanks @jenbreese! This looks good to me 🎉 

Feel free to merge it in when you're ready since I pre-approved 😄